### PR TITLE
Fixing issues of previous PRs

### PR DESCRIPTION
**Pull request by milanperera**

### CODE REVIEW

File: modules/mobile-agents/android/client/client/src/main/java/org/wso2/emm/agent/services/LocalNotification.java [hunk 1]
```diff
@@ -58,7 +58,7 @@ public static void startPolling(Context context) {
 	}
 
 	public static void stopPolling(Context context) {
-		if (localNoticicationInvoked == true) {
+		if (localNoticicationInvoked) {
 			localNoticicationInvoked = false;
 			Intent alarm = new Intent(context, AlarmReceiver.class);
 			PendingIntent sender = PendingIntent.getBroadcast(context, REQUEST_CODE, alarm, 0);
```
Comment (prabathabey) [inline modules/mobile-agents/android/client/client/src/main/java/org/wso2/emm/agent/services/LocalNotification.java]: What does "0" refer to? If this is some default value kinda thing, then you might have to define a constant and use upon it to be able to improve maintainability of the code.

Comment (milanperera) [inline modules/mobile-agents/android/client/client/src/main/java/org/wso2/emm/agent/services/LocalNotification.java]: Fixed it in this PR

File: modules/mobile-agents/android/client/client/src/main/java/org/wso2/emm/agent/RegistrationActivity.java [hunk 2]
```diff
@@ -198,7 +184,8 @@ public void onReceiveAPIResult(Map<String, String> result, int requestCode) {
 				responseStatus = result.get(Constants.STATUS);
 				Preference.putString(context, resources.getString(R.string.shared_pref_regId), info.getDeviceId());
 				if (Constants.Status.SUCCESSFUL.equals(responseStatus)) {
-					if(Preference.getString(context, context.getResources().getString(R.string.shared_pref_notifier)).trim().equals(Constants.NOTIFIER_GCM)){
+					if(Preference.getString(context, context.getResources().
+							getString(R.string.shared_pref_notifier)).trim().equals(Constants.NOTIFIER_GCM)){
 						registerGCM();
 					}else{
```
Comment (prabathabey) [inline modules/mobile-agents/android/client/client/src/main/java/org/wso2/emm/agent/RegistrationActivity.java]: There seems to be a formatting issue in this particular line. Even though it doesnt appear to be something you'd added, might be good to fix it.

File: modules/mobile-agents/android/client/client/src/main/java/org/wso2/emm/agent/api/Root.java [hunk 3]
```diff
@@ -89,7 +89,7 @@ public boolean checkRootBySuperUserApk() {
 	 */
 	public boolean checkRootBySuAccess() {
 		if (new ShellExecutor().executeCommand(SU_CHECK_COMMAND) != null) {
-			if (Constants.DEBUG_MODE_ENABLED){
+			if (Constants.DEBUG_MODE_ENABLED) {
 				Log.d(TAG, "su command enabled");
```
Comment (prabathabey) [inline modules/mobile-agents/android/client/client/src/main/java/org/wso2/emm/agent/api/Root.java]: Wrong grammar. The log line should ideally be "su command is enabled"

File: modules/mobile-agents/android/client/iDPProxy/src/main/java/org/wso2/emm/agent/proxy/AccessTokenHandler.java [hunk 4]
```diff
@@ -143,8 +139,7 @@ protected void onPostExecute(String result) {
                         editor.putString(Constants.DATE_LABEL, strDate);
                         editor.commit();
 
-                        identityProxy.receiveAccessToken(responseCode, Constants.SUCCESS_RESPONSE,
-                                token);
+                        identityProxy.receiveAccessToken(responseCode, Constants.SUCCESS_RESPONSE, token);
                     } catch (JSONException e) {
                         Log.e(TAG, "Invalid JSON format." + e);
```
Comment (prabathabey) [inline modules/mobile-agents/android/client/iDPProxy/src/main/java/org/wso2/emm/agent/proxy/AccessTokenHandler.java]: We don't seem to log the exception trace in this particular line. Again, even though this doesn't appear to be something that you'd added, we'd have to fix it.

**Pull request merged**